tests/keras/preprocessing/test_image.py

```diff
@@ -90,6 +90,55 @@ def test_img_flip(self):
                 assert ((potentially_flipped_x == x).all() or
                         (potentially_flipped_x == flip_axis(x, col_index)).all())
 
+    def test_dual_generators(self):
```

Comment:
My logic is that this test will break if any future PR break's repeatability, if you have time could you expand on why you don't think that's worth a test? In the mean time I'll remove it.